doc/news/changes/minor/20191102Wollner

```diff
@@ -0,0 +1,6 @@
+Changed: Fixed the computation of the FiniteElement.degree in
```

Comment:
```suggestion
Changed: Fixed the computation of the finite element degree in
```
I don't think doxygen will render this correctly so lets avoid code.

Reply:
Fine with me, I have updated the file.